### PR TITLE
initial version dashboard section

### PR DESCRIPTION
**Pull request by edewit**
## Motivation

[marvel](https://marvelapp.com/prototype/baei9db/screen/73638816)

## Brief Description

2 versions one on master and one empty for the other realms

## Verification Steps

click on dashboard in the left navigation bar

## Checklist:

- [x] Code has been tested locally by PR requester
- [x] User-visible strings are using the react-i18next framework (useTranslation)
- [x] Help has been implemented
- [x] axe report has been run and resulting a11y issues have been resolved
- [] Unit tests have been created/updated
- [x] Formatting has been performed via prettier/eslint
- [x] Type checking has been performed via 'yarn check-types'

## Additional Notes

image
image

### CODE REVIEW

File: src/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+
+  return (
+    <>
+      <ViewHeader
+        titleKey="dashboard"
+        subKey="last refresh"
+        subKeyLinkProps={{ title: "refresh", href: "hello" }}
+      />
+      <PageSection>
+        <Split hasGutter>
```
Comment (srambach) [inline src/dashboard/Dashboard.tsx:75]: To add the correct responsive behavior, you can use a nested grid with these values.
```
<Grid hasGutter>
  <GridItem span={12} lg={8}>
    <Grid hasGutter>
      <GridItem span={12} lg={4}>server info
      </GridItem>
      <GridItem span={12} lg={4}>server time
      </GridItem>
      <GridItem span={12} lg={4}>server uptime
      </GridItem>
      <GridItem span={12}>profile
      </GridItem>
      <GridItem span={12}>memory
      </GridItem>
    </Grid>
  </GridItem>
  <GridItem span={12} lg={4}>system
  </GridItem>
</Grid>
```

**Comment by srambach** [comment]
FYI, I left some comments on the Marvel because the spacing of the system info does not match the default spacing for a `dl` and I want to know if that was intentional.

File: src/PageNav.tsx [hunk 2]
```diff
@@ -89,6 +88,9 @@ export const PageNav: React.FunctionComponent = () => {
               )}
             </DataLoader>
           </NavList>
+          <NavList>
+            <LeftNav title="dashboard" path="/" />
+          </NavList>
```
Comment (srambach) [inline src/PageNav.tsx:93]: This should be a `NavGroup` instead of `NavList` to give proper spacing.

**Comment by edewit** [comment]
@ssilvert is this okay?
image

**Comment by ssilvert** [comment]
@edewit I think that will do fine for now.  I'll give it a try when you fix the conflicts.

**Review by mfrances17 — COMMENTED** [review]
Code LGTM, well done. I'd like to see it working before approving so once conflicts are resolved I'll test.

**Review by mfrances17 — APPROVED** [review]
LGTM, tested locally once conflicts were resolved and looks good

**Review by ssilvert — APPROVED** [review]

**Pull request merged**